harness/determined/common/storage/base.py

```diff
@@ -5,6 +5,12 @@
 from typing import Any, Dict, Iterator, Optional, Union
 
 
+class NoCloudAccess(Exception):
```

Comment:
Don't call this `cloud` access, because someday shared_fs checkpoints could also be proxied through the master.

Reply:
`NoDirectAccess`?

Also, can you put it in `det.errors.NoDirectAccess`?

Reply:
Sure. If we put it in `det.errors`, might want to make it more specific, something like `NoDirectStorageAccess`?

Reply:
Sure, sounds good.